FloripaPublicTransportation/Base.lproj/LaunchScreen.storyboard

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="9532" systemVersion="15D21" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" initialViewController="01J-lp-oVM">
```

Comment:
Hello @gobetti I'm curious to see your reasoning behind having the launch screen exactly like the initial app screen, can you elaborate?